cmds/core/ls/ls_test.go

```diff
@@ -92,12 +99,14 @@ func TestLs(t *testing.T) {
 
 	// Create some files.
 	os.Create(filepath.Join(testDir, "f1"))
-	os.Create(filepath.Join(testDir, "f2"))
+	f2, _ := os.Create(filepath.Join(testDir, "f2"))
```

Comment:
this test is not great, in that it ignores errors, but adding more ignored errrors is not the way we want to go.

Reply:
sure, will update this